docs/overview-of-architecture-and-components.md

```diff
@@ -10,11 +10,11 @@ The sample solution code (available [here](https://github.com/mspnp/iot-journey/
 
 The data for all devices is directed towards a Windows Azure Event Hub. From here, the data can take several paths:
 
-1. The data can be used as input to a Azure Stream Analytics job which simply passes the data through to Blob storage, recording it in JSON format. This is the *hot path* that provides the least friction for the raw data, allowing it to be stored very quickly for later analysis. Operations staff are provided with a Hive query that they can run on an HDInsight cluster to take this data and generate a report listing each device and the number of events it generated.
+1. The data can be used as input to a Azure Stream Analytics job which simply passes the data through to Blob storage, recording it in JSON format. This path provides the least friction for the raw data, allowing it to be stored very quickly for later analysis. Operations staff are provided with a Hive query that they can run on an HDInsight cluster to take this data and generate a report listing each device and the number of events it generated.
```

Comment:
These events might not be "for later analysis". There are scenario where the data is simply stored for auditing and accountability.